sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/VirtualMachinesOperationsExtensions.cs

```diff
@@ -230,12 +233,15 @@ public static void Delete(this IVirtualMachinesOperations operations, string res
             /// <param name='vmName'>
             /// The name of the virtual machine.
             /// </param>
+            /// <param name='forceDeletion'>
+            /// Optional parameter to force delete virtual machines.(Feature in Preview)
+            /// </param>
             /// <param name='cancellationToken'>
             /// The cancellation token.
             /// </param>
-            public static async Task DeleteAsync(this IVirtualMachinesOperations operations, string resourceGroupName, string vmName, CancellationToken cancellationToken = default(CancellationToken))
+            public static async Task DeleteAsync(this IVirtualMachinesOperations operations, string resourceGroupName, string vmName, bool? forceDeletion = default(bool?), CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Breaking change - need to include an overload that does not contain the ForceDeletion parameter to maintain backward compatibility

Reply:
Again this file has been generated by autorest tool using the last published swagger json (i.e. 30-06-2020). If we are allowed to make changes to generated files, will make the changes. please confirm

Reply:
Whether this is generated by AutoRest or not, we need to prevent breaking changes.  In this case, this is a breaking change in the SDK, which can be easily fixed with a single overload.  You should add this overload, or you will need breakign change approval.

Reply:
To be clear on how to implement this:  This is an extension - you would define a new static class in the same namespace and simply add an extension method overload that matches the old pattern.

Reply:
Added customizations as suggested.